pkg/api/rest/rest.go

```diff
@@ -28,6 +28,25 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
 )
 
+//TODO:
+// Storage interfaces need to be separated into two groups; those that operate
+// on collections and those that operate on individually named items.
+// Collection interfaces:
+// (Method: Current -> Proposed)
+//    GET: Lister -> CollectionLister
```

Comment:
CollectionGetter

List vs Get is arbitrary - a sub resource collection list is a collection get, so let's not confuse it.